tests/load-tests/run-max-concurrency.sh

```diff
@@ -136,39 +170,55 @@ max_concurrency() {
         echo "Maximal allowed length of user prefix is 10 characters. The '$USER_PREFIX' length of ${#USER_PREFIX} exceeds the limit."
         exit 1
     else
-        output="$output_dir/load-tests.max-concurrency.json"
+        output="$OUTPUT_DIR/load-test.max-concurrency.json"
         IFS="," read -r -a maxConcurrencySteps <<<"$(echo "${MAX_CONCURRENCY_STEPS:-1\ 5\ 10\ 25\ 50\ 100\ 150\ 200}" | sed 's/ /,/g')"
-        maxThreads=${MAX_THREADS:-10}
-        threshold=${THRESHOLD:-300}
-        echo '{"startTimestamp":"'"$(date +%FT%T%:z)"'", "maxThreads": '"$maxThreads"', "maxConcurrencySteps": "'"${maxConcurrencySteps[*]}"'", "threshold": '"$threshold"', "maxConcurrencyReached": 0, "computedConcurrency": 0, "workloadKPI": 0, "endTimestamp": "", "errorsTotal": -1}' | jq >"$output"
+        maxThreads=${MAX_THREADS:-10}   # Do not go above this concurrency.
+        threshold_sec=${THRESHOLD:-300}   # In seconds. If KPI crosses this duration, stop.
+        threshold_err=${THRESHOLD_ERR:-10}   # Failure ratio. When crossed, stop.
+        echo '{"started":"'"$(date +%FT%T%:z)"'", "maxThreads": '"$maxThreads"', "maxConcurrencySteps": "'"${maxConcurrencySteps[*]}"'", "threshold": '"$threshold_sec"', "thresholdErrors": '"$threshold_err"', "maxConcurrencyReached": 0, "computedConcurrency": 0, "workloadKPI": 0, "ended": "", "errorsTotal": -1}' | jq >"$output"
         iteration=0
+
+        {
+        python3 -m venv venv
+        set +u
+        source venv/bin/activate
+        set -u
+        python3 -m pip install -U pip
+        python3 -m pip install -e "git+https://github.com/redhat-performance/opl.git#egg=opl-rhcloud-perf-team-core&subdirectory=core"
+        python3 -m pip install tabulate
+        python3 -m pip install matplotlib
```

Comment:
```suggestion
python3 -m pip install matplotlib
python3 -m pip install 'tenacity<8.4.0'
```

Recent changes to tenacity module cause failues - see https://github.com/jd/tenacity/issues/471